sips/sip-018/sip-018-signed-structured-data.md

```diff
@@ -0,0 +1,204 @@
+# Preamble
+
+SIP Number: 018
+
+Title: Signed Structured Data
+
+Author: Marvin Janssen <https://github.com/MarvinJanssen>
+
+Consideration: Technical
+
+Type: Standard
+
+Status: Accepted
```

Comment:
Point of order -- the status of this PR is "draft".

Reply:
The language in SIP-000 seemed unclear to me.
>- _Draft_. The SIP is still being prepared for formal submission. It does not yet
  have a SIP number.

Pending feedback, the SIP is considered sufficiently complete and thus submitted.

Will change to "Draft".